cg/services/orders/store_order_services/store_pacbio_order_service.py

```diff
@@ -0,0 +1,111 @@
+            samples=status_data["samples"],
+        )
+        return {"project": project_data, "records": new_samples}
+
+    @staticmethod
+    def order_to_status(order: OrderIn) -> dict:
+        """Convert order input to status for PacBio-only orders."""
+        status_data = {
+            "customer": order.customer,
+            "order": order.name,
+            "samples": [
+                {
+                    "application": sample.application,
+                    "capture_kit": sample.capture_kit,
+                    "comment": sample.comment,
+                    "data_analysis": sample.data_analysis,
+                    "data_delivery": sample.data_delivery,
+                    "name": sample.name,
+                    "priority": sample.priority,
+                    "sex": sample.sex,
+                    "tumour": sample.tumour,
+                    "volume": sample.volume,
+                }
+                for sample in order.samples
+            ],
+        }
+        return status_data
+
+    def _store_samples_in_statusdb(
+        self, customer_id: str, order: str, ordered: datetime, ticket_id: str, samples: list[dict]
+    ) -> list[Sample]:
+        """Store PacBio samples in the status database including case links and delivery"""
+        customer: Customer = self.status_db.get_customer_by_internal_id(
+            customer_internal_id=customer_id
+        )
+        new_samples = []
+        case: Case = self.status_db.get_case_by_name_and_customer(
+            customer=customer, case_name=ticket_id
+        )
+        submitted_case: dict = samples[0]
+        with self.status_db.session.no_autoflush:
+            for sample in samples:
+                new_sample = self.status_db.add_sample(
+                    name=sample["name"],
+                    sex=sample["sex"] or "unknown",
+                    comment=sample["comment"],
+                    internal_id=sample.get("internal_id"),
+                    order=order,
+                    ordered=ordered,
+                    original_ticket=ticket_id,
+                    priority=sample["priority"],
+                    tumour=sample["tumour"],
+                    capture_kit=sample["capture_kit"],
+                )
+                new_sample.customer = customer
+                application_tag: str = sample["application"]
+                application_version: ApplicationVersion = (
+                    self.status_db.get_current_application_version_by_tag(tag=application_tag)
+                )
+                new_sample.application_version = application_version
+                new_samples.append(new_sample)
+                if not case:
+                    case = self.status_db.add_case(
+                        data_analysis=Workflow(submitted_case["data_analysis"]),
+                        data_delivery=DataDelivery(submitted_case["data_delivery"]),
+                        name=ticket_id,
+                        panels=None,
```

Comment:
I do not think we need to specify panels if it is None.

Reply:
removed panels